migration-roadmap/infield-button.md

```diff
@@ -0,0 +1,414 @@
+ 					})}
+ 					?disabled=${isDisabled}
+ 					aria-haspopup="listbox"
+ 					type="button"
+ 					tabindex=${tabIndex}
+-					aria-label="add"
++					aria-label="minus"
+ 					>
+ 					<div class="${rootClass}-fill">
+ 						${Icon(
+ 							{
+ 								size,
+-								iconName: "ChevronUp75",
++								iconName: "Dash",
+ 								setName: "ui",
+ 								customClasses: [`${rootClass}-icon`],
+ 							},
+ 							context,
+ 						)}
+ 					</div>
+ 				</button>
+ 				<button
+ 					class=${classMap({
+ 						[rootClass]: true,
+ 						[`${rootClass}--size${size?.toUpperCase()}`]:
+ 							typeof size !== "undefined",
+-						[`${rootClass}--bottom`]: typeof position !== "end",
++						[`${rootClass}--quiet`]: isQuiet,
+ 						"is-hover": isHovered,
+ 						"is-active": isActive,
+-						"is-focus-visible": isFocused,
+ 						...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+ 					})}
+ 					?disabled=${isDisabled}
+ 					aria-haspopup="listbox"
+ 					type="button"
+ 					tabindex=${tabIndex}
+-					aria-label="add"
++					aria-label="add"
+ 				>
+ 					<div class="${rootClass}-fill">
+ 						${Icon(
+ 							{
+ 								size,
+-								iconName: "ChevronDown75",
++								iconName: "Add",
+ 								setName: "ui",
+ 								customClasses: [`${rootClass}-icon`],
+ 							},
+ 							context,
+ 						)}
+ 					</div>
+ 				</button>
+ 			</div>
+ 			`
+ 		: html`
+ 				<button
+ 					class=${classMap({
+ 						[rootClass]: true,
+ 						[`${rootClass}--size${size?.toUpperCase()}`]:
+ 							typeof size !== "undefined",
+-						[`${rootClass}--${position}`]: typeof position !== "undefined",
+ 						[`${rootClass}--quiet`]: isQuiet,
+ 						"is-hover": isHovered,
+ 						"is-active": isActive,
+-						"is-focus-visible": isFocused,
+ 						...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+ 					})}
+ 					?disabled=${isDisabled}
+ 					aria-haspopup="listbox"
+ 					type="button"
+ 					tabindex=${tabIndex}
+-				>
++					@click=${onclick}
++					role="presentation"
++				>
+ 					<div class="${rootClass}-fill">
+ 						${when(iconName, () =>
+ 							Icon(
+ 								{
+ 									size,
+-									iconName,
++									iconName: iconNameWithSize,
+ 									setName: iconSet,
+ 									customClasses: [`${rootClass}-icon`],
+ 								},
+ 								context,
+ 							),
+ 						)}
+ 					</div>
+ 				</button>
+ 			`;
+```
+
+</details>
+
+### CSS => SWC mapping
+
+| CSS selector                            | Attribute or slot      | Status                       |
+| --------------------------------------- | ---------------------- | ---------------------------- |
+| `.spectrum-InfieldButton`               | Base element           | Implemented                  |
+| `.spectrum-InfieldButton--quiet`        | `quiet` attribute      | Implemented                  |
+| `.spectrum-InfieldButton-fill`          | Fill container         | Implemented                  |
```

Comment:
I mentioned it on one of the other components, but we could add the actual element's class here too! 

Something like "Fill container; `.fill`"